src/merkledag.proto.js

```diff
@@ -1,3 +1,6 @@
+'use strict'
+
+module.exports = new Buffer(`
```

Comment:
this is 'ugly'.. Now I see how require worked.. this doesn't remove complexity, it even adds.